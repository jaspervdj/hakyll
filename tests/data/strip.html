<div>
    I'm so rich I have $$3.

    $rev("foo")$
    $-rev(rev("foo"))$

    $if(body)-$
    I have body
    $else-$
    or no
    $-endif-$

    $if(unbound)$
    should not be printed
    $endif$

    $-if(body)-$
    should be printed
    $-endif$

    <ul>
    $for(authors)-$
        <li>$name$</li>
    $endfor-$
    </ul>

    $for(authors)-$
        $name-$
        $sep$,
    $-endfor$

    $body$
</div>

