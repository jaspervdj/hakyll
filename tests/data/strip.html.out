<div>
    I'm so rich I have $3.

    ooffoo

    I have body
    should be printed

    <ul>
    <li>Jan</li>
    <li>Piet</li>
    </ul>

    Jan,Piet

    <p>This is an example.</p>
</div>

